impala/thrift/process_thrift.sh

```diff
@@ -13,23 +13,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if [ -z "$IMPALA_REPO" ]; then
-    echo "Need to set IMPALA_REPO"
+function die() {
+    echo $1 >&2
     exit 1
-fi
+}
 
-if [ -z "$IMPYLA_REPO" ]; then
-    echo "Need to set IMPYLA_REPO"
-    exit 1
-fi
+[ -n "$IMPALA_REPO" ] || die "Need to set IMPALA_REPO"
+[ -n "$IMPYLA_REPO" ] || die "Need to set IMPYLA_REPO"
+
+source $IMPALA_REPO/bin/impala-config.sh
 
```

Comment:
Can you add set -euo pipefail after source impala-config.sh . I was trying to run this and ran into issues when it kept running after failing to copy some files